<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPoint</class>
 <widget class="QWidget" name="WidgetPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPointName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pressure Sensor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSensor">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Shaft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditShaft">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCleanUp">
       <property name="text">
        <string>Clean-up...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCompute">
       <property name="text">
        <string>Compute...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Layer number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSelectLayer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewParams">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>125</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfos">
      <attribute name="title">
       <string>Infos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Notice</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditNotice"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Infos</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTableView" name="tableViewInfos"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Installation</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>1333</width>
                 <height>540</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="labelSchema">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDataArrayAndPlots">
      <attribute name="title">
       <string>Data arrays and plots</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
           <item>
            <widget class="QPushButton" name="pushButtonExportMeasures">
             <property name="text">
              <string>Export cleaned measures</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRawData">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Show Raw Data</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="tableViewDataArray">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBoxPress">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Pressure</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="pressVBox"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBoxTemp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Temperatures</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_23">
            <item>
             <layout class="QVBoxLayout" name="tempVBox"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFluxes">
      <attribute name="title">
       <string>Fluxes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="fluxesSplitterVertical">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="fluxesSplitterHorizLeft">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_3">
           <layout class="QVBoxLayout" name="waterFluxVBox"/>
          </widget>
          <widget class="QWidget" name="layoutWidget_4">
           <layout class="QVBoxLayout" name="totalFluxVBox"/>
          </widget>
         </widget>
         <widget class="QSplitter" name="fluxesSplitterHorizRight">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="advectiveFluxVBox"/>
          </widget>
          <widget class="QWidget" name="layoutWidget_2">
           <layout class="QVBoxLayout" name="conductiveFluxVBox"/>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTemperature">
      <attribute name="title">
       <string>Temperature</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QSplitter" name="tempSplitterVertical">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>10</number>
         </property>
         <widget class="QSplitter" name="tempSplitterHorizLeft">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>10</number>
          </property>
          <widget class="QWidget" name="topLeftBox" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <layout class="QVBoxLayout" name="topLeftVLayout">
              <item>
               <layout class="QGridLayout" name="quantilesLayout">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBoxDirectModel">
                  <property name="text">
                   <string>Direct model</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="1" column="1">
                 <widget class="QLabel" name="labelRMSETherm2">
                  <property name="text">
                   <string>RMSE: </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="labelRMSETherm1">
                  <property name="text">
                   <string>RMSE: </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="radioButtonTherm1">
                  <property name="text">
                   <string>Thermometer 1</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="radioButtonTherm3">
                  <property name="text">
                   <string>Thermometer 3</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="radioButtonTherm2">
                  <property name="text">
                   <string>Thermometer 2</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="labelRMSETherm3">
                  <property name="text">
                   <string>RMSE: </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="botLeftBox" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <layout class="QVBoxLayout" name="botLeftVLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QSplitter" name="tempSplitterHorizRight">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>10</number>
          </property>
          <widget class="QWidget" name="topRightBox" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <layout class="QVBoxLayout" name="topRightVLayout"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="botRightBox" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_21">
            <item>
             <layout class="QVBoxLayout" name="botRightVLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabdistribution">
      <attribute name="title">
       <string>Parameters distribution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Best Parameters (rounded)  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Bins :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderBins">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBins">
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRefreshBins">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QScrollArea" name="scrollArea_12">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_17">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1363</width>
             <height>401</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QSplitter" name="splitter_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBoxLog10K">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <layout class="QVBoxLayout" name="log10KVBox"/>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="groupBoxConductivity">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="title">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <layout class="QVBoxLayout" name="conductivityVBox"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea_15">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_19">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1363</width>
             <height>400</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QSplitter" name="splitter_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBoxPorosity">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_22">
                <item>
                 <layout class="QVBoxLayout" name="porosityVBox"/>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="groupBoxCapacity">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="title">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_24">
                <item>
                 <layout class="QVBoxLayout" name="capacityVBox"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
